libvast/vast/system/evaluator.hpp

```diff
@@ -58,6 +58,9 @@ struct evaluator_state {
   /// Points to the parent actor.
   evaluator_actor::pointer self;
 
+  /// Stores the actor for sendings results to.
```

Comment:
Comment seems to be wrongly copied from the `partition_client_actor` directly below?